modules/@apostrophecms/area/ui/apos/components/AposAreaMenu.vue

```diff
@@ -131,6 +131,7 @@ export default {
       }
       return {
         menuPlacement: 'bottom',
+        menuOffset: 15,
```

Comment:
Should this offset always be fixed at 15? if so it should come after the instance level options

Reply:
I'm doing it it here because we're setting pixel styles for it. 15 is already the default, so I'm restating that here to make sure it's not overridden to the point where it no longer matches the offset style.